app/src/main/java/org/p2p/wallet/auth/ui/phone/countrypicker/CountryCodePickerPresenter.kt

```diff
@@ -1,13 +1,14 @@
 package org.p2p.wallet.auth.ui.phone.countrypicker
 
 import kotlinx.coroutines.launch
-import org.p2p.wallet.auth.ui.phone.CountryCodeInteractor
 import org.p2p.wallet.auth.model.CountryCode
 import org.p2p.wallet.auth.model.CountryCodeItem
+import org.p2p.wallet.auth.ui.phone.CountryCodeInteractor
 import org.p2p.wallet.common.mvp.BasePresenter
 import org.p2p.wallet.utils.emptyString
 
 private const val DEFAULT_KEY = ""
```

Comment:
private const val DEFAULT_KEY = emptyString()

Reply:
`private const val` can't be a function result

Reply:
okey but sad =(